plugins/scaffolder/src/components/TaskNotFound/TaskNotFound.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { BackstageTheme } from '@backstage/theme';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  container: {
+    paddingTop: theme.spacing(24),
+    paddingLeft: theme.spacing(8),
+    [theme.breakpoints.down('xs')]: {
+      padding: theme.spacing(2),
+    },
+  },
+  title: {
+    paddingBottom: theme.spacing(2),
+    [theme.breakpoints.down('xs')]: {
+      fontSize: 32,
+    },
+  },
+  body: {
+    paddingBottom: theme.spacing(6),
+    [theme.breakpoints.down('xs')]: {
+      paddingBottom: theme.spacing(5),
+    },
+  },
+}));
+
+export const TaskNotFound = () => {
```

Comment:
no "someone dropped the mic" reuse here?

Reply:
We left this in for now since it's a tiny bit more descriptive saying task not found. We should extend the 404 page itself to allow for this.

Reply:
looked at the mic page, we should totally do that instead. Let me fix it